src/components/MemberCounter.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import { Add, Remove } from '@mui/icons-material';
+import { Button, Grid, Typography } from '@mui/material';
+import { useState } from 'react';
+
+const MemberCounter = ({ member }) => {
+  const [count, setCount] = useState(member);
+
+  const onDecrease = () => {
+    if (count > 1) {
+      setCount((prevCount) => prevCount - 1);
+    }
```

Comment:
```suggestion
    (count > 1) && setCount((prevCount) => prevCount - 1)
```
이것은 취향의 문제인데 요렇게는 어떠실까요??

Reply:
코드가 훨씬 깔끔해지겠네요 :)
꼼꼼한 리뷰 너무 감사해요 도르!!
grid와 flex 사용은 저희 얘기를 나눠서 그 외의 수정 사항 반영해서 커밋 올리겠습니다 👍